server/sandbox_run_linux.go

```diff
@@ -375,6 +378,30 @@ func (s *Server) runPodSandbox(ctx context.Context, req *pb.RunPodSandboxRequest
 			}
 			g.SetLinuxCgroupsPath(cgPath + ":" + "crio" + ":" + id)
 			cgroupParent = cgPath
+
+			// check memory limit is greater than the minimum memory limit of 4Mb
+			// expand the cgroup slice path
+			slicePath, err := systemd.ExpandSlice(cgroupParent)
```

Comment:
how would this work for the cgroupfs backend?

Reply:
It won't, that is why it is executed under the systemd block only.